pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test: TestCCLLogic_multi_region_default_primary_region failed

pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test.TestCCLLogic_multi_region_default_primary_region [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9701185?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9701185?buildTab=artifacts#/) on release-22.2 @ [87cf2591b682aeeeea38533cf5b395c896ee357d](https://github.com/cockroachdb/cockroach/commits/87cf2591b682aeeeea38533cf5b395c896ee357d):


```
=== RUN   TestCCLLogic_multi_region_default_primary_region
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/b18615a2a85b82f2dc4668f3366969a7/logTestCCLLogic_multi_region_default_primary_region1250509262
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
[09:10:07] rng seed: 8044100012923309188

    logic.go:3025: 
        
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3406/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:23: CREATE DATABASE db
        expected success, but found
        (42602) region "invalid-region-name" does not exist
        region_util.go:105: in CheckClusterRegionIsLive()
    logic.go:2729: 
         pq: region "invalid-region-name" does not exist
[09:10:07] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3406/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region with config multiregion-9node-3region-3azs-no-los: 6 tests, 2 failures
    logic.go:3798: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3406/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:28: error while processing
    logic.go:3798: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3406/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:28: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b18615a2a85b82f2dc4668f3366969a7/logTestCCLLogic_multi_region_default_primary_region1250509262
--- FAIL: TestCCLLogic_multi_region_default_primary_region (4.70s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_multi_region_default_primary_region.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-27177

pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test.TestCCLLogic_multi_region_default_primary_region [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10170929?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10170929?buildTab=artifacts#/) on release-22.2 @ [c910f01100ba731e4b30a75d63431f19a026101d](https://github.com/cockroachdb/cockroach/commits/c910f01100ba731e4b30a75d63431f19a026101d):


```
=== RUN   TestCCLLogic_multi_region_default_primary_region
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/b18615a2a85b82f2dc4668f3366969a7/logTestCCLLogic_multi_region_default_primary_region3037909861
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
[08:51:59] rng seed: 7798552564153704216

    logic.go:3025: 
        
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3631/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:23: CREATE DATABASE db
        expected success, but found
        (42602) region "invalid-region-name" does not exist
        region_util.go:106: in CheckClusterRegionIsLive()
    logic.go:2729: 
         pq: region "invalid-region-name" does not exist
[08:51:59] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3631/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region with config multiregion-9node-3region-3azs-no-los: 6 tests, 2 failures
    logic.go:3798: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3631/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:28: error while processing
    logic.go:3798: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3631/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/multiregion-9node-3region-3azs-no-los/multiregion-9node-3region-3azs-no-los_test_/multiregion-9node-3region-3azs-no-los_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region:28: too many errors encountered, skipping the rest of the input
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b18615a2a85b82f2dc4668f3366969a7/logTestCCLLogic_multi_region_default_primary_region3037909861
--- FAIL: TestCCLLogic_multi_region_default_primary_region (7.64s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_multi_region_default_primary_region.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region
+++ b/pkg/ccl/logictestccl/testdata/logic_test/multi_region_default_primary_region
@@ -20,7 +20,7 @@ SET CLUSTER SETTING sql.defaults.primary_region = 'ap-southeast-2'
 statement error PRIMARY REGION must be specified if REGIONS are specified
 CREATE DATABASE db REGIONS "us-east1"
 
-query T noticetrace
+query T noticetrace retry
 CREATE DATABASE db
 ----
 NOTICE: defaulting to 'WITH PRIMARY REGION "ap-southeast-2"' as no primary region was specified

